docs/calendar.md

```diff
@@ -347,6 +347,38 @@ Temporal.Calendar.from('iso8601').fields(['month', 'day']);
 ```
 <!-- prettier-ignore-end -->
 
+### calendar.**mergeFields**(_fields_: object, _extraFields_: object) : object
```

Comment:
I don't think any cases for any of the ICU calendars would require merging fields to know what the output context is; the logic depends only on the fields present on the input. In the `role: 'merge'` logic in #1245, do you think there's any use case that requires knowing the output context for merging?

Reply:
Nope, I'm not aware of any. But we previously named a bunch of methods "dateXxx" for future compatibility reasons.  Wanted to make sure that those reasons don't apply here. If not, then name is fine as-is.

Reply:
My reasoning is that this can be `mergeFields` for the same reason `fields` isn't `dateFields`; it will work for both date and time fields if time calendars are added in the future.